backend/MASZ/Dtos/VerifiedEvidence/CreateVerifiedEvidenceDto.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace MASZ.Dtos.VerifiedEvidence
+{
+    public class CreateVerifiedEvidenceDto
+    {
+        [Required(ErrorMessage = "Channel ID is required")]
+        [RegularExpression(@"^\d{17,19}$", ErrorMessage = "the channel id can only consist of digits and be between 17 and 19 characters long")]
+        public string ChannelId { get; set; }
+        [Required(ErrorMessage = "Message ID is required")]
+        [RegularExpression(@"^\d{17,19}$", ErrorMessage = "the message id can only consist of digits and be between 17 and 19 characters long")]
+        public string MessageId { get; set; }
```

Comment:
```suggestion
        [Required(ErrorMessage = "Channel ID is required")]
        public ulong ChannelId { get; set; }
        [Required(ErrorMessage = "Message ID is required")]
        public ulong MessageId { get; set; }
```

we dont check on length since this is very weird to do. it may eventually overflow and nobody thinks about it :P
Furthermore, you can use `ulong` as type here and then ASP.NET does the type convertion and checks for us.

Reply:
Didn't know it did that, pretty handy. 

As for the length check yeah I guess it would need to be maintained every time discord changes their id length, which they have done twice now. I guess it exists to avoid errors from discord's api, but since the frontend doesn't really distinguish where the error occurred it can be removed. 

In the future it could be maintained better if there were constants somewhere in the codebase that define the minimum and maximum lengths of discord ids, but it's optional.